main/exercice/overview.php

```diff
@@ -155,7 +155,7 @@
     $btn_class = '';
 }
 
-if (!empty($attempts) && $visible_return['value'] == true) {
+if (!empty($attempts)) {
```

Comment:
This may seem to be a little dramatic change, but all in all the results must be shown depending on the 'results_disabled' test property.